ci_framework/roles/libvirt_manager/tasks/deploy_layout.yml

```diff
@@ -101,6 +98,12 @@
       }}
 
 - name: Create and run VMs
+  when:
```

Comment:
May make sense to add an assertion if the target is given but the host is not part of the inventory with a clean and clear failure?

Reply:
for now it won't fail. Not sure we want to go that way - we may have one variable file shared across multiple env, and then rely on the inventory only to deploy what we need. Might be something to consider, but it may create more issues than wanted.